assets/styles/parts/utils/_shared-variables.scss

```diff
@@ -6,4 +6,15 @@ All variables are available here:
 https://github.com/infinum/eightshift-frontend-libs/blob/develop/styles/scss/_core.scss
 */
 
-$base-col-number: global-settings(maxCols);
+$base-col-number: global-settings(esMaxCols);
```

Comment:
If it is available in global settings, shouldn't it be also available as a global variable?